BatteryCondition.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+
+namespace BatteryManagement
+{
+    internal class BatteryCondition
+    {
+        public static void CheckChargeRate(float chargeRate)
+        {
+            Constants.BatteryStates batteryChargeRate =CheckBatteryCondition(chargeRate, Constants.minChargeRate, Constants.maxChargeRate);
+            BatteryStatus.DisplayBatteryCondition("Charge Rate", batteryChargeRate);
+        }
+
+        public static void CheckChargeState(float soc)
+        {
+            Constants.BatteryStates batteryChargeState =CheckBatteryCondition(soc, Constants.minStateOfCharge, Constants.maxStateOfCharge);
+            BatteryStatus.DisplayBatteryCondition("State of charge", batteryChargeState);
+        }
+
+        public static void CheckTemperature(float temperature)
+        {
+            Constants.BatteryStates batteryTemperature =CheckBatteryCondition(temperature, Constants.minTemperature, Constants.maxTemperature);
+            BatteryStatus.DisplayBatteryCondition("Temperature", batteryTemperature);
+        }
+        public static Constants.BatteryStates CheckBatteryCondition(float ActualValue, float minValue, float maxValue)
+        {
```

Comment:
separate each battery status condition check to different functions.
argument names must be in camel convention
When all the members of class is static mark class as static

Reply:
battery condition separated for min and max case.
Argument names modified.